extensions/amp-position-observer/0.1/amp-position-observer.js

```diff
@@ -353,14 +353,17 @@ export class AmpVisibilityObserver extends AMP.BaseElement {
     let scene;
     if (this.targetId_) {
       scene = user().assertElement(
+        // TODO: In friendly iframe, getAmpDoc() incorrectly fetches the parent
+        // doc. Need to change it to return the FIE AMP doc.
         this.getAmpDoc().getElementById(this.targetId_),
```

Comment:
better to fix the issue using `this.win.document` and have a TODO to revert it back to `this.getAmpDoc`